src/components/FileList.tsx

```diff
@@ -79,7 +79,8 @@ export const CONTEXT_COMMANDS: ContextCommands = {
     ContextCommandIDs.gitCommitAmendStaged,
     ContextCommandIDs.gitFileHistory
   ],
-  unmodified: [ContextCommandIDs.gitFileHistory]
+  unmodified: [ContextCommandIDs.gitFileHistory],
+  unmerged: [ContextCommandIDs.gitFileOpen]
```

Comment:
I think raw file with conflict is bad for most users. So let's turn on only the merge widget for now:

```suggestion
  unmerged: [ContextCommandIDs.gitFileDiff]
```